src/portal-components/DetailsPage.tsx

```diff
@@ -18,19 +18,25 @@ import { SynapseConfig } from 'types/portal-config'
 import { DetailsPageProps, RowSynapseConfig } from 'types/portal-util-types'
 import './DetailsPage.scss'
 import injectPropsIntoConfig from './injectPropsIntoConfig'
-import { cloneDeep } from 'lodash'
 import { ExternalFileHandleLink } from 'synapse-react-client/dist/containers/ExternalFileHandleLink'
 import { BarLoader } from 'react-spinners'
+import SvgIcon from "@material-ui/icons/ExploreOutlined"
+import ExploreOutlinedIcon from '@material-ui/icons/ExploreOutlined'
+import {LockedFacet} from "synapse-react-client/dist/containers/QueryWrapper"
+import {ReactComponent as StudySVG} from "synapse-react-client/dist/assets/icons/study2.svg"
+import {ReactComponent as DataSummarySVG} from "synapse-react-client/dist/assets/icons/Data2.svg"
+
```

Comment:
I have to direct import the svg. These icons are not available from Material UI. And our ICON component uses image tag <img src="...svg path"> . The image tag prevents me from customizing the fill colors of the svg icons.